client/packages/coldchain/src/Sensor/api/operations.graphql

```diff
@@ -1,4 +1,4 @@
-fragment LocationRow on LocationNode {
+fragment SensorLocationRow on LocationNode {
```

Comment:
This is just renamed, as the codegen complained about having two different types called `LocationRow` (it was fine when they were exactly the same, but now one of them is fetching additional info)